Deeploy/CommonExtensions/TypeCheckers/SignPropTypeChecker.py

```diff
@@ -60,15 +60,22 @@ def typeInferOutput(self, ctxt: NetworkContext, node: gs.Node,
 
         inputs = [ctxt.lookup(inputNode.name) for inputNode in node.inputs]
         outputs = [ctxt.lookup(outputNode.name) for outputNode in node.outputs]
-
+        print(f"[DEBUG - typeInferOutput] Node: {node.name}")
+        for i, inp in enumerate(inputs):
+            print(f"  Input {i}: name={inp.name}, has _signed={hasattr(inp, '_signed')}, has nLevels={hasattr(inp, 'nLevels')}")
         signProp = all([hasattr(_input, "_signed") and hasattr(_input, "nLevels") for _input in inputs])
-
+        print(f"  signProp = {signProp}")
         if signProp:
             nLevels = self._inferNumLevels(inputs, operatorRepresentation)
             signedness = self._inferSignedness(inputs, operatorRepresentation)
+            print(f"  inferred nLevels = {nLevels}")
+            print(f"  inferred signedness = {signedness}")
 
-            for obj, nLevels, sign in zip(outputs, nLevels, signedness):
-                obj.nLevels = nLevels
+            for obj, nl, sign in zip(outputs, nLevels, signedness):
```

Comment:
Was this causing issues, because of the double usage of nLevels? if so, maybe just keep it as nLevel or something similar, to keep it clear for future people that might work on this.

Also, don't forget to remove the debugging prints.